<launch>

    <arg name="image_path" default="/home/ysw/catkin_ws/src/visualizer/src/posenet_pytorch/data"/>
    <arg name="metadata_path" default="/home/ysw/catkin_ws/src/visualizer/src/posenet_pytorch/data/dataset_test.txt"/>

    <node pkg="rviz"        type="rviz"                 name="rviz"                 args="-d $(find visualizer)/launch/rviz_config.rviz" />
    <node pkg="visualizer"  type="rviz_node.py"         name="rviz_node"            output="screen"/>
    <node pkg="visualizer"  type="test.py"              name="posenet_node"         output="screen" args="--image_path $(arg image_path) --metadata_path $(arg metadata_path)">
    </node>
    <node pkg="visualizer"  type="data_publish_node.py" name="data_publish_node"    output="screen"/>
    <node pkg="visualizer"  type="image_node.py"        name="image_node"           output="screen"/>
    
</launch>